<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlignmentPlot</class>
 <widget class="QWidget" name="AlignmentPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="aData_Label">
        <property name="text">
         <string>Data A</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QComboBox" name="aData_ComboBox"/>
        </item>
        <item>
         <widget class="QSpinBox" name="aAlignment_SpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="bData_Label">
        <property name="text">
         <string>Data B</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QComboBox" name="bData_ComboBox"/>
        </item>
        <item>
         <widget class="QSpinBox" name="bAlignment_SpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="TelemetryPlot" name="plot" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TelemetryPlot</class>
   <extends>QWidget</extends>
   <header>GoProOverlay/graphics/TelemetryPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
